src/app/Table.elm

```diff
@@ -11,93 +11,114 @@ import Html exposing (..)
 import Html.Attributes exposing (..)
 import Html.Events exposing (..)
 import Types exposing (PlayerID, PlayerName)
-import TableMsg exposing (Msg(..))
 import Dice
+import Identicon exposing (identicon)
 
 
 type alias DiceCount =
     Int
 
 
 type alias TablePlayer =
-    { id : PlayerID
+    { id : Int
```

Comment:
This was my attempt at having this module have its own private types, but no... it's probably a mistake (I *am* importing `PlayerID`, weird that it changed). I'll adjust. Thanks for the pickup.